### ISSUE TITLE
No indication of where other party has read to

### ISSUE THREAD

**Issue by jimeh** [yakyak/yakyak#87]
Official clients indicates upto where the other parties in the conversation has read to. YakYak doesn't seem to have any such indicator as far as I've seen.

**Comment by AdrianoCahete** [comment]
:+1:

**Comment by x-tk** [comment]
That'd be nice to have.

**Comment by vandasian** [comment]
:+1: This is the main thing missing.

**Comment by vandasian** [comment]
#55

**Comment by geekgonecrazy** [comment]
Poked around a bit.  Looks like the information is definitely available.  So would just be a matter of displaying it.  Which admittedly looks like the hardest part :grin:

**Comment by averissimo** [comment]
where did you find it available?

**Comment by averissimo** [comment]
Partial implementation of this in 1.4.0-beta3

see #451 and activate option via view -> experimental

The library that yakyak uses does not support (yet) having this information when opening a conversation, only after first interaction

**Comment by averissimo** [comment]
can anyone test a build with this fixed?

@silveran

reply with operating system

**Comment by Silveran** [comment]
It works like charm for me on MacOS Sierra

**Comment by averissimo** [comment]
the implementation is not complete, you only get the seen after the other user interacts or opens the conversation.

doesn't it?

**Comment by Silveran** [comment]
Just tried it, I confirm what you said.
After launch the app, no read notification before the other open the discussion on his app.

**Comment by averissimo** [comment]
@silveran had problems trying to only show the last conversation seen (instead of all others)

here it is: https://github.com/averissimo/yakyak/releases/tag/sandbox

**Comment by Silveran** [comment]
I got an issue with the zip, unable to unzip, it gives me an `error 20 - Not a directory`

**Comment by averissimo** [comment]
sorry, tried to be a smart ass and install the new library manually -_-

uploading now an extractable version

**Comment by Silveran** [comment]
Seems to work well now
<img width="163" alt="screenshot 2016-12-05 a 01 53 23" src="https://cloud.githubusercontent.com/assets/10518880/20870758/b2ada326-ba8d-11e6-8b97-15c9b3d61ce4.png">

**Comment by averissimo** [comment]
So cool! thanks for testing.

**Comment by davibe** [comment]
this will be available in 1.4.1 (you can test it in the pre-release, or just wait a little bit)

**Comment by hnilsen** [comment]
Running 1.4.4, this seems to be missing. Has something changed in the API, or has the feature been removed again? Cannot find a way to activate it.
